packages/strapi-design-system/src/Badge/__tests__/Badge.e2e.js

```diff
@@ -1,13 +1,15 @@
-import { injectAxe, checkA11y } from 'axe-playwright';
+const { injectAxe, checkA11y } = require('axe-playwright');
 
-describe('Badge', () => {
-  beforeEach(async () => {
+const { test } = require('@playwright/test');
+
+test.describe('Badge', () => {
```

Comment:
some of the tests have `test.describe.parallel` and others only `test.describe`, is it wanted?